public/index.html

```diff
@@ -2,403 +2,1360 @@
-                </a>
-            </div>
-            <div class="mb-3 col-6 col-md-3 pics">
-                <img alt="Logo Virage Digital" class="img-fluid"
-                     src="build/front/images/clients/viragedigital.png">
-            </div>
-            <div class="mb-3 col-6 col-md-3 pics">
-                <a href="https://wertkt.com/" target="_blank">
-                    <img alt="Logo TkT" class="img-fluid"
-                         src="build/front/images/clients/tkt.png">
-                </a>
-            </div>
-            <div class="mb-3 col-6 col-md-3 pics">
-                <a href="https://www.norsys.fr/" target="_blank">
-                    <img alt="Logo Norsys" class="img-fluid"
-                         src="build/front/images/clients/norsys.png">
-                </a>
-            </div>
-            <div class="mb-3 col-6 col-md-3 pics">
-                <img alt="Logo Orange" class="img-fluid"
-                     src="build/front/images/clients/orange.png">
-            </div>
-            <div class="mb-3 col-6 col-md-3 pics">
-                <a href="https://www.fk-agency.com/" target="_blank">
-                    <img alt="Logo Fk Agency" class="img-fluid"
-                         src="build/front/images/clients/fk.png">
-                </a>
-            </div>
-
-        </div>
-
-    </section>
-
-    <hr>
-
-    <section class="section mb-5" id="team">
-
-        <h3 class="text-center title mt-5 pt-4 font-weight-bold">
-            <strong>L'équipe</strong>
-        </h3>
-        <p class="text-center w-responsive mb-5 pb-4 mx-auto">Cinq profils
-            complémentaires.</p>
-
-        <div class="row text-center">
-            <div class="card-group row justify-content-center">
-                <div class="col-lg-3 col-md-6 mb-4">
-                    <div class="card">
-                        <div class="avatar mx-auto p-3">
-                            <img alt="Alexandre Balmes"
-                                 class="rounded-circle img-fluid"
-                                 src="build/front/images/profilepic/alexandre.jpg">
+        <div id="projects" class="relative bg-white pt-16 pb-32 overflow-hidden">
+            <div class="relative">
+                <div class="lg:mx-auto lg:max-w-7xl lg:px-8 lg:grid lg:grid-cols-2 lg:grid-flow-col-dense lg:gap-24">
+                    <div class="px-4 max-w-xl mx-auto sm:px-6 lg:py-16 lg:max-w-none lg:mx-0 lg:px-0">
+                        <div>
+                            <div class="mt-6">
+                                <h2 class="text-3xl font-extrabold tracking-tight text-gray-900">Création d'un jeux en ligne</h2>
+                                <p class="mt-4 text-lg text-gray-500">
+                                    Création d'un serveur de jeux en Rust afin de gérer les nombreuses connexions simultanées
+                                    et la mise en place d'un protocole de communication client/serveur.
```

Comment:
```suggestion
                                    Création d'un serveur de jeux robuste et performant en Rust afin de gérer des milliers de connexions et parties simultanées. Création du client en React et mise en place d'un protocole de communication binaire client/serveur le plus léger possible.
```

Quelque chose comme ça